app/models/volunteer.rb

```diff
@@ -124,66 +129,77 @@ def has_main_region?
   end
 
   def main_region
-    self.regions[0]
+    regions[0]
```

Comment:
Looks like `.first` tacks a `LIMIT 1` on the end of the query, so probably slightly better. Will update 👍